packages/plugins/documentation/server/utils/builders/build-api-endpoint-path.js

```diff
@@ -97,7 +97,7 @@ const getPaths = ({ routeInfo, attributes, tag }) => {
       responses,
       tags: [_.upperFirst(tag)],
       parameters: [],
-      requestBody: {},
+      operationId: `${methodVerb}${_.upperFirst(routePath)}`,
```

Comment:
I'm not sure the `upperFirst` is working since the first character in the string is a `/`, but I'm wondering if it is necessary anyway? Couldn't we just leave the route path as it is?

Reply:
You're right, the `upperFirst` doesn't change the `routPath` at all. I removed it.
Thanks for your feedback.